Readme update

thatSFguy opened the pull request:
Updated the Readme & added the necessary boot loader files in zip format.

thatSFguy committed:
Updated Readme and added bootloader zip

thatSFguy committed:
fixed indentation

NomDeTom commented:
@thatSFguy I was coming here to change the instructions myself, but I see you've already got here.

With the bootloader update, you don't need the hex file - the UF2 file is all that is required for the update.

You could go further and delete the lines 29-31 in the readme, as well.

thatSFguy commented:
Hey @NomDeTom,  thanks for the feedback.  I got these nice nanos from Amazon and that's how I got it to update the bootloader.  If you have a UF2 file that just does it all, please share.  I got a couple of boards from @wbrucem and they have a different bootloader, so maybe he used the one you are talking about.

thatSFguy committed:
Update README.md

moved things around a little to section out the GUI method of flashing and the CLI way of flashing.

valzzu commented:
@thatSFguy if uf2 bootloader doesent work then u can use zip one with nrfutil, thats how i update most of them.
ive had few that needed to be flashed with swd since they didnt have usb bootloader.

thatSFguy commented:
> @thatSFguy if uf2 bootloader doesent work then u can use zip one with nrfutil, thats how i update most of them. ive had few that needed to be flashed with swd since they didnt have usb bootloader.

I think I got one without a bootloader as well. Let me see if I can find it and get it to work.  Thanks!
